composer.json

```diff
@@ -35,13 +35,13 @@
         "neos/utility-unicode": "*",
         "doctrine/orm": "^2.6",
         "ext-dom": "*",
-        "neos/twitter-bootstrap": "*",
+        "neos/twitter-bootstrap": "^3.0.6",
```

Comment:
Do we really need that here? Didn't we move the dependency to `neos/media-browser`? See: #4673

Reply:
Ahhh sorry. I was confused by the (none) package sorting here 🤦